libs/coin-modules/coin-polkadot/.unimportedrc.json

```diff
@@ -25,6 +25,7 @@
     "src/types/bridge.fixture.ts"
   ],
   "ignoreUnused": [
+    "@ledgerhq/live-env",
```

Comment:
If this is not use anymore can we juste remove it from package.json ? 
Need to be tested to be sure that it's not breaking anything